src/rgw/rgw_lua_utils.h

```diff
@@ -20,8 +21,10 @@ void lua_pushtime(lua_State* L, const CephTime& tp)
   lua_pushstring(L, buff);
 }
 
-// push std::string
-void lua_pushstring(lua_State* L, const std::string& str);
+static inline void pushstring(lua_State* L, std::string_view str)
```

Comment:
why is using a `string_view` better than `string&` ?

Reply:
`string_view` is a better read-only container for strings. We pass those around in Ceph with increasing frequency. If you use `const std::string&` then whenever you push a `std::string_view`, you'll get an (unnecssary) intermediate string created.